doc_user/UserManual.adoc

```diff
@@ -2650,6 +2650,8 @@ The functions available in this window (similar to other windows in the *_Types_
 * *[.underline]##D##elete*: delete an existing type (cannot be deleted if still used somewhere in the application).
 * *[.underline]##C##lose*: close the window.
 
+Deleted Medical Types will be marked as “Deleted” and will not be shown in the list of Medical Types in the application. User cannot mark a Medical Type as “Deleted” if it is still used somewhere in the application.
+
```

Comment:
I think the note should be armonized with the above buttons description, this is a suggestion: @dbmalkovsky wdyt?

```suggestion
* *[.underline]##D##elete*: delete an existing type. *
* *[.underline]##C##lose*: close the window.

*DELETION - Medical Types cannot be deleted if still used somewhere in the application (i.e. linked to previous data) but can be marked as “Deleted” and will not be shown in the list of Medical Types in the application. If a Medical Type has never been used or is not linked to any previous data, the deletion will be completed with the actual record removal.

```

Reply:
@mwithi your suggested changes sounds good.

Reply:
BTW, why is there no update of image148.png showing the new `Deleted` column?

Reply:
there is

Reply:
sorry; somehow I missed it